src/commands/setup-for-local-development.js

```diff
@@ -53,6 +53,10 @@ class SetupForLocalDevelopmentCommand extends BaseCommand {
               title: 'Register masternode',
               task: () => registerMasternodeTask(config),
             },
+            {
```

Comment:
Not sure we need the miner here.

Reply:
I added this because on line 64 I had to remove `isMinerEnabled: true` so it works with the new arguments taken by `startNodeTask`

Reply:
Yeah, but we don't need a miner for this command I believe :)

Reply:
Oh right :(... But shouldn't update config just to run this exact task. It seems we still need this option for `startNodeTask` just to change behavior for this specific call. So it should be like this there:

```js
if (isMinerEnabled === undefined) {
    isMinerEnabled = config.get('core.miner.enable');
}
```

Reply:
`registerMasternodeTask` already updates the config with the BLS key, so I think it doesn't matter if we also change the miner setting? And miner is defined as `false` in the system config, so we can't check for `undefined`...

Reply:
The register command supposed obtain and store/output key, but here it will be a side effect. I don't need the miner, then run this command, and the miner is enabled for some reason. It’s strange.

Reply:
Oh I understand now, the miner is needed while the `setup-for-local-development` task is running, but should not be in the config once setup is done, right? Changes done as you explained.